scopes/harmony/modules/cli/loader/loader.ts

```diff
@@ -0,0 +1,3 @@
+import loader from '@teambit/legacy/dist/cli/loader';
```

Comment:
we need to move the real loader here, we don't allow regular components (which are not aspects) to be dependent on @teambit/legacy

Reply:
we can't do that, because if there are 2 instances of the loader it creates bugs in the cli

Reply:
why can't we take the code from `src/cli/loader/loader.ts` into the loader here
and replace the code in the `src/cli/loader/loader.ts` to something like
`export loader as default from '@teambit/harmony.modules.cli.loader';`

And of course, add it to the package.jon once we export